db/schema.rb

```diff
@@ -116,14 +116,16 @@
     t.date "end_date"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "organization_id"
+    t.index ["organization_id"], name: "index_diaper_drives_on_organization_id"
```

Comment:
Same problem with unrelated schema changes.

Reply:
This is related.